lumicks/pylake/nb_widgets/tests/test_kymotracker_widget.py

```diff
@@ -0,0 +1,97 @@
+from lumicks.pylake.nb_widgets.kymotracker_widgets import KymoWidgetGreedy
+from matplotlib.testing.decorators import cleanup
+import numpy as np
+import pytest
+
+
+@cleanup
+def test_widget_open(kymograph):
+    KymoWidgetGreedy(kymograph, 1, use_widgets=False)
+
+
```

Comment:
Great set of tests! For the future, I propose telling a story for each block/function...makes for much easier reviewing. Something like 
`# Given a kymograph,  alternating pixel thresholds should result in a kymo widget with corresponding lines`

Reply:
Will do!